lib/mappers/pa_mapper.py

```diff
@@ -0,0 +1,122 @@
+from dplaingestion.selector import exists, getprop
+from dplaingestion.mappers.dublin_core_mapper import DublinCoreMapper
+from dplaingestion.utilities import iterify
+from dplaingestion.selector import exists, setprop
+from urlparse import urlparse
+from akara import logger
+
+
+class PAMapper(DublinCoreMapper):
+
+    dcmi_types = {
```

Comment:
@markbreedlove I can refactor the name but functionally this works the same as a List.